helm/prometheus-agent/charts/prometheus-agent/templates/prometheus.yaml

```diff
@@ -8,6 +8,12 @@ metadata:
   name: {{ include "name" . }}
   namespace: {{ .Release.Namespace }}
 spec:
+  {{- if .Values.discovery.tuned }}
```

Comment:
I don't see value with this `tuned` property.
Why not relying only on `.Values.discovery.reloadInterval`?